### PR TITLE
Dark Mode

### LINKED ISSUE #8: Dark mode

### PR DESCRIPTION
**Pull request by filiptronicek**
Fix #8

### CODE REVIEW

File: header.php [hunk 1]
```diff
@@ -24,6 +24,6 @@
     </nav>
     <dark-mode-toggle
       id="dark-mode-toggle"
-      appearance="switch"
+      appearance="toggle"
```
Comment (filiptronicek) [inline header.php:27]: I suggest we add the permanent attribute to preserve the user selection upon reload and other navigation.
```suggestion
      appearance="toggle"
      permanent="true"
```

Comment (Vilican) [reply header.php:27]: Make sure that user has consented to use of cookies if we are going to remember his selection this way.

Comment (filiptronicek) [reply header.php:27]: @Vilican this is handled by localStorage, not cookies 👀 
https://github.com/GoogleChromeLabs/dark-mode-toggle/blob/a8f4d3c7746ccba60dae86d7fb44b49e90cfb4e1/src/dark-mode-toggle.mjs#L19

Comment (filiptronicek) [reply header.php:27]: Anyways I found that the info would have to identify the user in some way, but the selection is never sent out from the client, so correct me if I'm wrong, but I think we don't need it here

**Pull request closed without merging**